docs/v0.7/en/greptimecloud/integrations/emqx.md

```diff
@@ -0,0 +1,24 @@
+# EMQX Platform
+
+[EMQX Platform](https://www.emqx.io/) is an MQTT Gateway, designed to handle
+massive amounts of IoT device connections and message traffic, making it a
+popular choice for building large-scale IoT applications. It has built-in
+support for GreptimeDB as a data integration. By adding GreptimeDB as a Data
+Persistent sink, you can ingest EMQX messages into GreptimeDB automatically.
+
+You will need to follow these steps for your complete IoT data link, from MQTT
+to database:
+
+- Sign up your account on [EMQX Platform](https://www.emqx.io/)
+- Create a **Dedicated Instance** and wait for it's up and running
+- Setup Private Link or NAT Gateway for your deployment so it has internet
+  access
+- Go to **Data Integrations** and find **GreptimeDB**
+- Configure your **GreptimeDB** connector using following information
+  - Server host: `<host>:4001`
```

Comment:
When the EMQ cloud updates the greptimedb ingester, we must change the port to `5001`.Just a note.